static/bidder-params/unruly.json

```diff
@@ -1,13 +1,20 @@
 {
-    "$schema": "http://json-schema.org/draft-04/schema#",
-    "title": "Unruly Adapter Params",
-    "description": "A schema which validates params accepted by the Unruly adapter",
-    "type": "object",
-    "properties": {
-      "siteid": {
-        "type": "integer",
-        "description": "ID for publisher site"
-      }
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Unruly Adapter Params",
+  "description": "A schema which validates params accepted by the Unruly adapter",
+  "type": "object",
+  "properties": {
+    "siteid": {
+      "type": "integer",
+      "description": "ID for publisher site"
     },
-  "required": ["siteid"]
+    "siteId": {
+      "type": "integer",
+      "description": "ID for publisher site"
```

Comment:
Which casing do you prefer to use? We have a convention of ending the description of that property with `, preferred.` as a hint to publishers.

Reply:
Do you mean to add a `,` after the end of the `description`?

Reply:
I think @SyntaxNode is wondering if you could update the description of the bidder param you prefer between `siteId` and `siteid`

So if `siteId` is the preferred casing, you can update its description to be like `"ID for publisher site, preferred"`